index.html

```diff
@@ -105,6 +105,7 @@ <h2 id="contribute">Contribute</h2>
 					<li><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
 					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
 					<li><a href="http://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
+					<li><a href="https://github.com/git-for-windows/git/releases/">All Releases</a></li>
```

Comment:
I am fine with "All Releases", but I do wish to have a commit message that conforms better to our existing history. That is:

1. commit subject in imperative form: "Add an `All Release` link"
2. use the commit body to explain the rationale instead of leaving it empty.
3. Sign-off on the patch.

Thanks.